source/server/config_validation/server.cc

```diff
@@ -106,8 +100,8 @@ void ValidationInstance::shutdown() {
   // do an abbreviated shutdown here since there's less to clean up -- for example, no workers to
   // exit.
   thread_local_.shutdownGlobalThreading();
-  if (config_ != nullptr && config_->clusterManager() != nullptr) {
-    config_->clusterManager()->shutdown();
+  if (config_.clusterManager() != nullptr) {
```

Comment:
As I already mentioned in the other PR, this is almost certainly not correct. The nullptr check was present previously because shutdown() can be called before there is a config. The cluster manager is created during initialize(). I don't think there is a guarantee that the cluster manager exists here. Same applies in the non-validation server I think (or for sure that one, maybe not this one).